File: cpp/bc1962.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main()
{
	int count, ano, resY;
	cin >> count;
	
	for(int i = 0; i < count; i++)
	{	
		cin >> ano;
		
		resY = 2015-ano;
		
		resY > 0? cout << resY << " D.C." << endl : cout << abs(resY)+1 << " A.C." << endl;
		
	}
}

File: python/bc1962.py
count = int(input())

for c in range(0, count):

    ano = int(input())

    res = 2015 - ano

    #if res == 0: res += 1

    print(f"{res} D.C." if res > 0 else f"{abs(res)+1} A.C.")
